internal/kafka/internal/api/public/api/openapi.yaml

```diff
@@ -2035,6 +2053,15 @@ components:
           type: string
         max_connection_attempts_per_sec:
           type: integer
+        max_message_size:
+          description: Maximum message size
+          type: string
```

Comment:
I find it a bit confusing to have a K8s size string here and the bytes object in the other API returning this. Can't we have the same bytes object here?

Reply:
See https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1006#issuecomment-1121186214 comment